integration/test_valkey_search_acl.py

```
@@ -15,35 +15,85 @@ class TestCommandsACLs(ValkeySearchTestCaseBase):
         self, client: Valkey, cmd: List[str], should_access: bool
     ):
-        try:
+        if should_access:
+            try:
+                client.execute_command(*cmd)
+            except ResponseError as e:
+                # Non-ACL errors are acceptable here, such as missing indexes.
+                error = str(e).lower()
+                assert "no permissions" not in error
+                assert "permission to access" not in error
+            except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)